composer/workflows/trigger_response_dag.py

```diff
@@ -18,28 +18,17 @@
 import datetime
 
 import airflow
-from airflow.operators import bash_operator
+from airflow.operators.bash_operator import BashOperator
 
 
-default_args = {
-    'owner': 'Composer Example',
-    'depends_on_past': False,
-    'email': [''],
-    'email_on_failure': False,
-    'email_on_retry': False,
-    'retries': 1,
-    'retry_delay': datetime.timedelta(minutes=5),
-    'start_date': datetime.datetime(2017, 1, 1),
-}
-
 with airflow.DAG(
         'composer_sample_trigger_response_dag',
-        default_args=default_args,
+        start_date=datetime.datetime(2021, 1, 1),
```

Comment:
Please add a timezone if it makes sense in this context. 😄

Reply:
It does not matter for this use case because the DAG is triggered by GCF, but good lookin out! This date update just makes this look up to date :)